pandas/core/groupby/ops.py

```diff
@@ -424,6 +424,7 @@ def _call_cython_op(
                     mask=mask,
                     result_mask=result_mask,
                     is_datetimelike=is_datetimelike,
+                    **kwargs,
```

Comment:
Why is kwargs needed here?

Reply:
To pass skipna to idxmin/idxmax.

Reply:
Ah OK and I guess not all these accept skipna (IIRC there was that related issue related `skipna`, but forgot if it was for groupby ops or not)

Reply:
Yea - I'm planning on turning #56939 into a tracking issue on adding skipna consistently throughout groupby. Can include removing these kwargs as part of that.